xml/System.IO.Compression/BrotliEncoder.xml

```diff
@@ -47,10 +47,19 @@
         <Parameter Name="window" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="quality">To be added.</param>
-        <param name="window">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="quality">A number representing quality of the Brotli compression. 0 is the minimum (no compression), 11 is the maximum.</param>
+        <param name="window">A number representing the encoder window bits. The minimum value is 10, and the maximum value is 24.</param>
+        <summary>Initializes an instance of <see cref="T:System.IO.Compression.BrotliEncoder" /> class using the specified quality and window.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="quality" /> is not between the minimum value of 0 and the maximum value of 11.
+
+-or-
+
+<paramref name="window" /> is not between the minimum value of 10 and the maximum value of 24.
+        </exception>
+        <exception cref="T:System.InvalidOperationException">Quality and window cannot be changed at the current encoder state.</exception>
+        <exception cref="T:System.ObjectDisposedException">Cannot access a closed encoder.</exception>
```

Comment:
Removed based on @rpetrusha 's above comment. Will investigate if it's truly possible to throw this exception in the constructor.
```suggestion
```